pandas/plotting/_core.py

```diff
@@ -2818,18 +2818,42 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw an area plot of the input series using matplotlib.
+
+        An area plot displays graphically quantitative data.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Coordinates for X point.
+        y : label or position, optional
+            Coordinates for Y point.
+        kwds : optional
             Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
 
         Returns
         -------
-        axes : matplotlib.AxesSubplot or np.array of them
+        matplotlib.AxesSubplot or numpy.ndarray
+            Area plot generated.
+
+        See Also
+        --------
+        pandas.DataFrame.plot : Draw a plot using matplotlib.
+
+        Examples
+        --------
+        .. plot::
+            :context: close-figs
+
+            This example draws an area plot based on the length and width of
```

Comment:
1) I think is better to put the text before the start of the `plot` block, and leave the block only for the code that constructs the plot.
2) The example this sentence talks about doesn't correspond to the dataframe below.
3) Don't write in 3rd person `This example does this`. Better write directly in imperative or present form, like `Draw an area plot based on` or `We can draw an area plot based on...`, but IMO mentioning `this example` is somewhat redundant when we are already in the examples section. :)